install/kubernetes/cilium/templates/cilium-operator/deployment.yaml

```diff
@@ -196,9 +196,9 @@ spec:
           mountPath: /var/lib/cilium/bgp
           readOnly: true
         {{- end }}
-        {{- if .Values.operator.resources }}
+        {{- with .Values.operator.resources }}
         resources:
-          {{- toYaml .Values.operator.resources | trim | nindent 10 }}
+          {{- toYaml . | trim | nindent 10 }}
```

Comment:
This change is supposed to be part of the 1st commit but is part of the 3rd?

Reply:
The 1st commit also contains `priorityClassName` changes, so I decided to squash 3rd commit into 1st commit.
I also update 1st commit message to match w/ this PR description.